### PR TITLE
Weeb Patchu 3.3

### PR DESCRIPTION
**Pull request by BlueLightning42**
fixed spaces -> tabs

added banana

### CODE REVIEW

File: Brain/ArchiBoT/Custom/blueweeb.siml [hunk 1]
```diff
@@ -1,126 +1,141 @@
...
-          <Item>Anime Weeb Trash.</Item>
-          <Item>Why don't you ask blue weeb :3</Item>
-        </Random>
-      </Response>
-    </Model>
+	<Model>
+		<Pattern>
+			<Item>:CAVEMANBANANA:</Item>
+			<Item>ːCAVEMANBANANAː</Item>
+			<Item> % BANANA % </Item>
```
Comment (Thielak) [inline Brain/ArchiBoT/Custom/blueweeb.siml]: That's going to intercept anything that includes the word banana

File: Brain/ArchiBoT/Custom/blueweeb.siml [hunk 2]
```diff
@@ -1,126 +1,141 @@
...
-      </Response>
-    </Model>
+	<Model>
+		<Pattern>
+			<Item>^TABS OR ^SPACES</Item>
+			<Item>^SPACES OR ^TABS</Item>
+		</Pattern>
+		<Response>
+			<Random>
+			<Item>TAB MASTERRACE!</Item>
```
Comment (Thielak) [inline Brain/ArchiBoT/Custom/blueweeb.siml]: This still only has only one item inside, making the Random and Item tags useless

File: Brain/ArchiBoT/Custom/blueweeb.siml [hunk 3]
```diff
@@ -1,126 +1,141 @@
...
-          <Item>Moshi moshi you kawai kyuties, you're all my nakama and I lub you.</Item>
-          <Item>Hai Hai! Arigato for the best anime ever made railgun!</Item>
-        </Random>
-      </Response>
-    </Model>
-	
+		</Pattern>
+		<Response>
+			<Random>
+			<!-- Please feel free to remove anything you don't think is good or add anythign to the following -->
```
Comment (Thielak) [inline Brain/ArchiBoT/Custom/blueweeb.siml]: Rip indentations forever

File: Brain/ArchiBoT/Custom/blueweeb.siml (hunk 1 above)
Comment (BlueLightning42) [reply Brain/ArchiBoT/Custom/blueweeb.siml]: yes

I know :3 

its also a shitty reply tho idk what to make it say

File: Brain/ArchiBoT/Custom/blueweeb.siml (hunk 2 above)
Comment (BlueLightning42) [reply Brain/ArchiBoT/Custom/blueweeb.siml]: I actually saw that myself...and fixed it in the next patchu

Comment (BlueLightning42) [reply Brain/ArchiBoT/Custom/blueweeb.siml]: *it had other shit in and then when I deleted all of it I forgot to remove the random as well

File: Brain/ArchiBoT/Custom/blueweeb.siml (hunk 3 above)
Comment (BlueLightning42) [reply Brain/ArchiBoT/Custom/blueweeb.siml]: TAB MASTER RACE

File: Brain/ArchiBoT/Custom/blueweeb.siml (hunk 1 above)
Comment (Thielak) [reply Brain/ArchiBoT/Custom/blueweeb.siml]: I told you because you're supposed to fix it :doubt:

**Comment by BlueLightning42** [comment]
I'm just happy I can constantly push changes and all my mistakes are squashed :3

**Pull request merged**